include/picongpu/fields/FieldJ.tpp

```diff
@@ -194,7 +194,7 @@ SimulationDataId FieldJ::getUniqueId( )
     return getName( );
 }
 
-HDINLINE
+HINLINE
```

Comment:
It should not be required here at all. In case a definition is separate from a declaration, the declaration defines that a function is `inline` and at definition it does not matter whether you add it or not.

Reply:
I kept it because of all other methods are defined as `HINLINE` too